README.md

```diff
@@ -111,13 +124,14 @@ $ forge build
 #### 2. Validate the deterministic addresses
 
 ```shell
-forge script 'script/Deploy.s.sol:DeployScript' -vvvv --rpc-url "$RPC_URL" --private-key "$PK" "hooks.cow.eth" --sig "run(string)"
+forge script 'script/DeployAndRecord.s.sol:DeployAndRecordScript' -vvvv --rpc-url "$RPC_URL" "hooks.cow.eth" --sig "run(string)"
```

Comment:
nit: I think this shell command is easier to read if you add the argument after the `--sig` (since we read left to right).
Also, would make the sig to go right after the script name.

`forge script 'script/DeployAndRecord.s.sol:DeployAndRecordScript' --sig "run(string)" "hooks.cow.eth"--rpc-url "$RPC_URL" -vvv`